includes/payment-methods/class-wc-stripe-payment-request.php

```diff
@@ -522,10 +526,12 @@ public function scripts() {
 				'unknown_shipping' => __( 'Unknown shipping option "[option]".', 'woocommerce-gateway-stripe' ),
 			),
 			'checkout'        => array(
-				'url'            => wc_get_checkout_url(),
-				'currency_code'  => strtolower( get_woocommerce_currency() ),
-				'country_code'   => substr( get_option( 'woocommerce_default_country' ), 0, 2 ),
-				'needs_shipping' => WC()->cart->needs_shipping() ? 'yes' : 'no',
+				'url'               => wc_get_checkout_url(),
+				'currency_code'     => strtolower( get_woocommerce_currency() ),
+				'country_code'      => substr( get_option( 'woocommerce_default_country' ), 0, 2 ),
+				'needs_shipping'    => WC()->cart->needs_shipping() ? 'yes' : 'no',
+				// Defaults to 'required' to match how core initializes this option.
+				'needs_payer_phone' => 'required' === get_option( 'woocommerce_checkout_phone_field', 'required' ),
```

Comment:
TIL

Reply:
Yeah, it took a while to figure that out too! The [WP Customizer documentation](https://developer.wordpress.org/themes/customize-api/customizer-objects/#settings) is particularly wordy, and there's no explicit section on how to retrieve values (that information is at the bottom of the section I linked to called "Settings"). It also doesn't say that the values are only added to the DB **once changed**, which is why it's important to include the default value in the `get_option` call.

Once I had all that figured out, I then had to dig through the  WC Core code to find the correct [option name](https://github.com/woocommerce/woocommerce/blob/788969d2bbfba8f7e177b83504e70e98bc1babdf/includes/customizer/class-wc-shop-customizer.php#L685-L700).

It's simple once you know it though 😛